src/base/memory_monitor_cl.hpp

```diff
@@ -28,6 +28,17 @@
 
 #include "memory_monitor_common.h"
 
-// XXX: will be added print function
+#define CONVERT_TO_KB_SIZE(size) ((size) / 1024)
 
+constexpr char module_names[MMON_MODULE_LAST][DB_MAX_IDENTIFIER_LENGTH] =
+{
+  /* XXX: this dummy modules will be changed when heap module is registered */
```

Comment:
```suggestion
  /* TODD: this dummy modules will be changed when heap module is registered */
```
or TODO(xxx). "TODO" is usually highlighted in many text editors, so easy to notice And definitive :).

Reply:
In my text editor(vim) also highlightes XXX too, but I think you're right. I change it.